lib/wallet/wallet.js

```diff
@@ -86,6 +86,13 @@ class Wallet extends EventEmitter {
       consensus.MAX_BLOCK_OPENS * bidsPerAuction * bidsCacheSizeInBlocks;
     this.sendBidResults = new LRU(capacity);
 
+    const updatesCacheSizeInBlocks = 20;
+    const updatesPerAuction = 3; // average in March 2020 is ~3.1
```

Comment:
We're using this for registers so technically this value should be 1 (there's at most 1 register per auction).